packages/perseus-editor/src/widgets/interactive-graph-editor.tsx

```diff
@@ -73,23 +73,73 @@ const FieldLabel = (props: {children: string}) => {
 type Props = {
     apiOptions: APIOptionsWithDefaults;
 
+    /**
+     * The labels for the x and y axes.
+     */
     labels: ReadonlyArray<string>;
+    /**
+     * The range of the graph in the x and y directions.
+     */
     range: [Range, Range]; // x, y
```

Comment:
One thing we can do now in Typescript is add names to the tuple items. I don't think we can use them when referring to the tuple items, but I think it might help with documentation. 

Not requiring this in this PR, but you can if you like the idea. 

https://blog.logrocket.com/exploring-use-cases-typescript-tuples/#what-are-named-tuples

```suggestion
    range: [x: Range, y: Range]; // x, y
```